contracts/allocators/LUSDAllocatorRescue.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.8.10;
+
+import "../interfaces/IAllocator.sol";
+import "./interfaces/IWETH.sol";
+import "../types/BaseAllocator.sol";
+
+error LUSDAllocator_InputTooLarge();
+error LUSDAllocator_TreasuryAddressZero();
+
+/**
+ *  Contract deploys LUSD from treasury into the liquity stabilty pool. Each update, rewards are harvested.
+ *  The allocator stakes the LQTY rewards and sells part of the ETH rewards to stack more LUSD.
+ *  This contract inherits BaseAllocator is and meant to be used with Treasury extender.
+ */
+contract LUSDAllocatorV2R is BaseAllocator {
+    using SafeERC20 for IERC20;
+
+    /* ======== STATE VARIABLES ======== */
+    address public immutable wethAddress = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public immutable lqtyTokenAddress = 0x6DEA81C8171D0bA574754EF6F8b412F2Ed88c54D;
+
+    /**
+     * @notice tokens in AllocatorInitData should be [LUSD Address]
+     * LUSD Address (0x5f98805A4E8be255a32880FDeC7F6728C6568bA0)
+     */
+    constructor(AllocatorInitData memory data) BaseAllocator(data) {}
+
+    // Send ETH to an address
+    function sendETH(address to_) external {
+        _onlyGuardian();
+
+        payable(to_).transfer(address(this).balance);
```

Comment:
It's better to use a low-level call since if you send it to a smart contract wallet or a contract, it could get bricked in the future if the gas cost of op codes change. Transfer only forwards 2300 gas.

`payable(to_).call{value: address(this).balance}("");`

Reply:
cool, updated